x-pack/plugins/infra/server/lib/alerting/metric_threshold/preview_metric_threshold_alert.test.ts

```diff
@@ -82,6 +90,8 @@ describe('Previewing the metric threshold alert type', () => {
         },
         lookback: 'h',
         alertInterval: '1m',
+        alertThrottle: '1m',
+        alertOnNoData: true,
       });
```

Comment:
Do we have tests where the throttle is different than the interval, to test the changes in this ticket?

Reply:
Just added a test for the throttling logic!